### DISCUSSION TITLE
How does this compare to Voidtools Everything?

### DISCUSSION THREAD

**Discussion by endolith** [Flow-Launcher/Flow.Launcher#1234] [Q&A] [answered]
With *Everything*, I hit Win+Esc, a window pops up, I start typing, and if the thing I want pops up, I press Enter twice to launch it.  

It only searches file names by default, but it does have a (much slower) `content:` search function for searching inside files (and a bunch of other modifiers I never learned).

Flow Launcher seems to have more functionality?  Is it equally fast?

**Comment by taooceros** [comment, marked as answer]
We use windows index by default for file search (and content search), and we do have an everything plugin to utilize everything to search (which means it is basically equally fast except for some trivial cost).
